engines/director/sprite.cpp

```diff
@@ -502,7 +498,7 @@ void Sprite::setCast(CastMemberID memberID, bool replaceDims) {
 
 	_castId = memberID;
 	_cast = _movie->getCastMember(_castId);
-	//As QDShapes don't have an associated cast, we must not change their _SpriteType.
+	// As QDShapes don't have an associated cast, we must not change their _SpriteType.
```

Comment:
This is the only change in this file that makes sense and should stay. The rest to revert.